.github/workflows/synthetics-stack-e2e-init.yml

```diff
@@ -0,0 +1,36 @@
+---
+#
+# This workflow sets the status check `synthetics-stack-e2e` instantly to 'failure'
+# the synthetics-stack-e2e.yml is triggered with a PR comment and is responsible for setting it to success or failure.
+#
+name: synthetics
+
+on:
+  pull_request_target:
+    paths:
+      - "x-pack/plugins/synthetics/**"
```

Comment:
Is there a situation where this workflow runs and you do not want `synthetics-check-run` to run? I'm wondering if it makes sense to trigger the `e2e tests` automatically and the developer wouldn't need to comment.

Reply:
Hi @Ikuni17, the synthetics stack e2e test is a long running test and we only want to run the test before merging.

to be sure: @shahzad31 can you please confirm this?

Or do you think we could also just run it on every push?

Reply:
indeed, we want to run it on demand. not on every push.

Reply:
Thanks for explanation.